package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.0.1",
   "description": "A simple atom package to run a UNIX filter on current text",
   "keywords": [],
+  "activationCommands": {
+    "atom-workspace": ["unix-filter:run"]
```

Comment:
There are two available commands though, so I reckon both should be listed here.

Reply:
Why are they here after all? I don't know why... but I added this one in case short key needs it. Why add the second one?

Reply:
See here: https://flight-manual.atom.io/hacking-atom/sections/package-word-count/#packagejson

TL;DR Atom won't activate the package until one of `activationCommands` is executed.